src/reactionsystem.jl

```diff
@@ -76,25 +76,106 @@ Base.Sort.defalg(::ReactionComplex) = Base.DEFAULT_UNSTABLE
 
 #! format: off
 # Internal cache for various ReactionSystem calculated properties
+# All related functionality is in the "network_analysis.jl" file. However, this must be declared
+# here as the structure is part of the `ReactionSystem` structure.
 Base.@kwdef mutable struct NetworkProperties{I <: Integer, V <: BasicSymbolic{Real}}
+    """Flag which is switched to `true` once any field is updated."""
     isempty::Bool = true
+    """
+    The reaction network's net stoichiometry matrix. It is an MxN matrix where M is its number of 
+    species and N is its number of reactions. Element i,j is the net stoichiometric change to the
+    i'th species as a result of the j'th reaction.
+    """
     netstoichmat::Union{Matrix{Int}, SparseMatrixCSC{Int, Int}} = Matrix{Int}(undef, 0, 0)
+    """
+    The reaction network's conservation law matrix. It is an MxN matrix where M is its number of 
+    conservation laws and N its number of species. Element i,j is the coefficient of species 
+    j in the i'th conservation law.
+    """
     conservationmat::Matrix{I} = Matrix{I}(undef, 0, 0)
     col_order::Vector{Int} = Int[]
```

Comment:
`col_order` was the only one which I felt unsure of, could you confirm what it is @isaacsas